frontend/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@shopgate/pdp-native-share",
+  "version": "1.0.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "scripts": {
+    "lint": "eslint --parser babel-eslint --ignore-path ../.gitignore --ignore-path .eslintignore --ext .js --ext .jsx .",
+    "test": "jest",
+    "test:watch": "npm run test -- --watch",
+    "coverage": "jest --coverage",
+    "coverage:watch": "npm run coverage -- --watch",
+    "dummy": ""
+  },
+  "devDependencies": {
+    "@shopgate/eslint-config": "*",
+    "@shopgate/pwa-common": "5.10.0-alpha.1",
+    "@shopgate/pwa-common-commerce": "5.10.0-alpha.1",
+    "@shopgate/pwa-core": "5.10.0-alpha.1",
+    "@shopgate/pwa-unit-test": "5.10.0-alpha.1",
+    "@shopgate/pwa-ui-ios": "5.10.0-alpha.1",
+    "@shopgate/pwa-ui-material": "5.10.0-alpha.1",
+    "babel-core": "^6.26.3",
+    "babel-plugin-dynamic-import-webpack": "^1.0.2",
+    "babel-plugin-lodash": "^3.3.2",
+    "babel-plugin-module-resolver": "^3.1.0",
+    "babel-plugin-react-transform": "^3.0.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-inline-elements": "^6.22.0",
+    "babel-plugin-transform-react-jsx-source": "^6.22.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-latest": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "coveralls": "^3.0.2",
+    "eslint": "^4.17.0",
+    "eslint-import-resolver-babel-module": "4.0.0",
+    "glamor": "^2.20.40",
+    "jest": "^23.5.0",
+    "prop-types": "^15.6.0",
+    "react": "~16.3.2",
+    "react-dom": "~16.3.2",
+    "react-helmet": "^5.1.3",
+    "react-hot-loader": "4.0.1",
+    "react-redux": "^5.0.5",
+    "redux-mock-store": "^1.5.3",
+    "reselect": "^3.0.1"
+  },
+  "peerDependencies": {
```

Comment:
the peerDependencies here are wrong. We need to add `^` in front. Like "**^**5.10.0-alpha.1"